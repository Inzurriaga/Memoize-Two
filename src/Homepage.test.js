import React from 'react';
import { shallow } from 'enzyme';
import HomePage from "./Homepage"

const mockUser = {"name":"gabriel","password":"gabriel","completed":[1,2,5]}
const mockUsers = [{"name":"duy","password":"duy","completed":[1,2,5]},{"name":"gabriel","password":"gabriel","completed":[1,2,5]},{"name":"isaac","password":"isaac","completed":[1,2]}]

describe("HomePage", ()=> {
  let wrapper;
  beforeEach(() => {
      wrapper = shallow(
        <HomePage users={mockUsers}
                    user={mockUser}/>
      )
  })

  it("should have a snapshot", () => {
      expect(wrapper).toMatchSnapshot();
  })
})