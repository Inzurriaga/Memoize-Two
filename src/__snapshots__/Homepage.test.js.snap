// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should have a snapshot 1`] = `
<div>
  <PlayerStatus
    user={
      Object {
        "completed": Array [
          1,
          2,
          5,
        ],
        "name": "gabriel",
        "password": "gabriel",
      }
    }
  />
  <RankContainer
    users={
      Array [
        Object {
          "completed": Array [
            1,
            2,
            5,
          ],
          "name": "duy",
          "password": "duy",
        },
        Object {
          "completed": Array [
            1,
            2,
            5,
          ],
          "name": "gabriel",
          "password": "gabriel",
        },
        Object {
          "completed": Array [
            1,
            2,
          ],
          "name": "isaac",
          "password": "isaac",
        },
      ]
    }
  />
</div>
`;
